reportportal/values.yaml

```diff
@@ -157,6 +157,8 @@ minio:
   region:
   accesskey: <minio-accesskey>
   secretkey: <minio-secretkey>
+  bucketPrefix: ""
+  defaultBucketName: ""
```

Comment:
Here add another parmater: integrationSaltPath

Reply:
added it.